### ISSUE TITLE
Add instructions / dockerize

### ISSUE THREAD

**Issue by openp2pdesign** [Make-IT-TR/TechRadar#12]
Please add some instructions on how to run it locally, or even better, add a Dockerfile for the whole app.

**Comment by openp2pdesign** [comment]
@damylen this is still missing, please add instructions / dockerization or it's very hard or impossible for me to work on the interface!

From what I've seen, some packages were missing and I added them with yarn (`yarn add d3 marked katex circular-json`), and I've found this as a way for starting the app:

```
cd tr-server
npm install
cd..

cd tr-app
npm install
yarn install
npm start
```

But the Radar cannot be seen:

![screen shot 2017-08-01 at 19 39 21](https://user-images.githubusercontent.com/1458229/28838629-32978f82-76f1-11e7-85c7-77bf0689bf15.png)

**Comment by openp2pdesign** [comment]
With the current Dockerfile I get:

```
$ docker run -p 3000:80 tr
npm info it worked if it ends with ok
npm info using npm@5.3.0
npm info using node@v8.4.0
npm info lifecycle skeleton-typescript-webpack@1.0.0~prestart: skeleton-typescript-webpack@1.0.0
npm info lifecycle skeleton-typescript-webpack@1.0.0~start: skeleton-typescript-webpack@1.0.0

> skeleton-typescript-webpack@1.0.0 start /tr-app
> nps

sh: nps: not found
npm info lifecycle skeleton-typescript-webpack@1.0.0~start: Failed to exec start script
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! skeleton-typescript-webpack@1.0.0 start: `nps`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the skeleton-typescript-webpack@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2017-09-05T14_58_34_756Z-debug.log
```

**Comment by openp2pdesign** [comment]
If I add nps to the server, I get this error while building the container:

```
Step 8/12 : RUN npm --prefix ./tr-server install ./tr-server
 ---> Running in 5ab9f8206a42
npm info it worked if it ends with ok
npm info using npm@5.3.0
npm info using node@v8.4.0
npm info lifecycle tr-server@1.0.0~preinstall: tr-server@1.0.0
npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but npm-shrinkwrap.json was generated for lockfileVersion@0. I'll try to do my best with it!
http fetch GET 200 https://registry.npmjs.org/jQuery/-/jQuery-1.7.4.tgz 246ms
npm WARN tr-server@1.0.0 No description
npm WARN tr-server@1.0.0 No repository field.

npm ERR! opts.mkdirpAsync is not a function

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2017-09-05T15_05_24_266Z-debug.log
The command '/bin/sh -c npm --prefix ./tr-server install ./tr-server' returned a non-zero code: 1
```

**Comment by openp2pdesign** [comment]
Nevermind, fixed with 0a69dafb31add2430ef70902937d4e2b14283a31
